src/imagealternatetext/imagealternatetext.js

```diff
@@ -38,6 +38,13 @@ export default class ImageAlternateText extends Plugin {
 	init() {
 		this._createButton();
 
+		// Push button to default image toolbar if one exists.
```

Comment:
"if the image toolbar feature was loaded" or something like that would be more precise.